Scripts/models/logit.py

```diff
@@ -426,10 +441,32 @@ def calc_individual_mode_prob(self, is_car_user, zone):
                     else:
                         mode_exps[mode] *= math.exp(b["car_users"][1])
             mode_expsum += mode_exps[mode]
+            if mode == "car":
+                car_expsum += mode_exps[mode]
+            else:
+                sust_expsum += mode_exps[mode]
         probs = numpy.empty(len(modes))
         for i, mode in enumerate(modes):
             probs[i] = mode_exps[mode] / mode_expsum
-        return probs
+        # utils to money
+        logsum = numpy.log(mode_expsum)
+        sust_logsum = numpy.log(sust_expsum)
+        car_logsum = numpy.log(car_expsum)
+        b = self._get_cost_util_coefficient()
+        if isinstance(b, tuple):
```

Comment:
I have used try-except clauses for these, as they are faster for the standard case, and might be considered more pythonical (https://docs.python.org/3.5/glossary.html#term-eafp).